tests/config/test_default.py

```diff
@@ -0,0 +1,19 @@
+from unittest import TestCase
+
+from frontrunner_sdk.config import DEFAULT
+
+
+class TestDefaultFrontrunnerConfig(TestCase):
```

Comment:
Can we call this test "TestFrontrunnerConfig"? The test case name can stay the same.